must-gather/collection-scripts/gather_collection_resources

```diff
@@ -68,4 +68,7 @@ do
     check_collector_persistence $pod
     oc -n openshift-logging exec -- ls -l /var/lib/fluentd/clo_default_output_es > $collector_folder/$pod.es-buffers.txt
     oc -n openshift-logging exec -- ls -l /var/lib/fluentd/retry_clo_default_output_es > $outdir/$pod.buffers.es-retry.txt
+    echo "$pod" >> $collector_folder/all-pod-buffer-size.txt
```

Comment:
this would be more appropriately named something related to output buffers.  These are buffers associated with fluentd pods but the buffers are for each output configured in the CLF.

Reply:
Yes, hence I have used a "*" to capture buffer sizes since fluentd mostly errors out with "buffer" related messages and this is the first thing that can be checked. Using wildcard would capture all the files under `/var/lib/fluentd` accordingly.

Reply:
I have renamed the output file, please review and let me know if you have any other suggestions.